packages/clay-drop-down/src/drilldown/Menu.tsx

```diff
@@ -88,55 +93,59 @@ const DrilldownMenu: React.FunctionComponent<IProps> = ({
 									onClick,
 									symbol,
 									title,
+									type,
 									...other
 								},
 								j
-							) => (
-								<li key={`${j}-${title}`}>
-									<LinkOrButton
-										{...other}
-										buttonDisplayType="unstyled"
-										className={classNames(
-											'dropdown-item',
-											className
-										)}
-										data-testid={`menu-item-${title}`}
-										onClick={(
-											event: React.SyntheticEvent
-										) => {
-											if (onClick) {
-												onClick(event);
-											}
+							) =>
+								type === 'divider' ? (
+									<Divider key={`${j}-divider`} />
+								) : (
```

Comment:
A good strategy to avoid conditionals for cases like this is to create a dictionary of valid types and access it to render the respective components, we can decrease some of the noise, we do the same for `DropDownWithItems` but we don't need to worry about that now since we support only one `type`.

Reply:
I thought about using a look up table, but for a simple case like this it seemed like overkill, that said let me have second look. I don't think there's any rush here, and we can probably agree quickly.

Reply:
Yes it can be a possibility too, well I'm fine if we go like this, we can change that in the future if we have more use cases.